app/(authed)/success/error.tsx

```
@@ -0,0 +1,49 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { ROUTES } from "@/lib/constants";
+import { AlertCircle } from "lucide-react";
+import Link from "next/link";
+import { useEffect } from "react";
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)